spec/01-unit/05-utils_spec.lua

```diff
@@ -676,4 +676,27 @@ describe("Utils", function()
       end, "scale must be equal or greater than 0")
     end)
   end)
+
+  describe("get_mime_type()", function()
+    it("with valid mime types", function()
+      assert.equal("application/json; charset=utf-8", utils.get_mime_type("application/json"))
+      assert.equal("application/json; charset=utf-8", utils.get_mime_type("application/json; charset=utf-8"))
+      assert.equal("application/json; charset=utf-8", utils.get_mime_type("application/*"))
+      assert.equal("application/json; charset=utf-8", utils.get_mime_type("application/*; charset=utf-8"))
+      assert.equal("text/html; charset=utf-8", utils.get_mime_type("text/html"))
+      assert.equal("text/plain; charset=utf-8", utils.get_mime_type("text/plain"))
+      assert.equal("text/plain; charset=utf-8", utils.get_mime_type("text/*"))
+      assert.equal("text/plain; charset=utf-8", utils.get_mime_type("text/*; charset=utf-8"))
+      assert.equal("application/xml; charset=utf-8", utils.get_mime_type("application/xml"))
+      assert.equal("", utils.get_mime_type("application/grpc"))
+    end)
+
+    it("with unsupported or invalid mime types", function()
+      assert.equal("application/json; charset=utf-8", utils.get_mime_type("audio/*", true))
+      assert.equal("application/json; charset=utf-8", utils.get_mime_type("text/css"))
```

Comment:
Please add a line for the "default" content type, which is a bit special since it is defined as `CONTENT_TYPE_DEFAULT`:
```suggestion
      assert.equal("application/json; charset=utf-8", utils.get_mime_type("text/css"))
      assert.equal("application/json; charset=utf-8", utils.get_mime_type("default"))
```